exercises/20_loops/group_1_level_1_100.ipynb

```diff
@@ -133,7 +133,15 @@
      "is_executing": false
     }
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
Please don't commit outputs. Clear all outputs prior to commiting.

Reply:
fixed